src/Console/Command/HelpCommand.php

```diff
@@ -269,6 +269,17 @@ public static function getHelpCopy()
 
 Where ``$COMMIT_RANGE`` is your range of commits, e.g. ``$TRAVIS_COMMIT_RANGE`` or ``HEAD~..HEAD``.
 
+Additionally, you might configure some caching to speed up your continuous builds, e.g. for GitHub Actions, this can be achieved using:
+
+    - name: "Cache results of PHP CS Fixer"
```

Comment:
dont know how to notate yaml here, which will be converted into proper RST formatting

Reply:
Let's try with `.. code-block:: yaml` and don't forget ti regenerate README.